core/pkg/server/handler.go

```diff
@@ -1265,6 +1269,47 @@ func (h *Handler) handleWandbConfigParameters(record *service.Record) {
 	h.sendRecord(record)
 }
 
+func (h *Handler) writeAndSendConfigFile(record *service.LaunchConfigFileParameterRecord) {
```

Comment:
can this be implemented in the launch package instead? at the very least a separate file; the `handler.go` file does far too much.

Reply:
`launch` is imported by `server` so I don't think we can access file handler. Another option would be for the launch python SDK could save this file to the run before it publishes the `LaunchConfigFileParameterRecord`... in which case I will just rip this out of the PR. I think I am good with that.

Reply:
I moved it to the launch package 👍🏻